pyls/plugins/definition.py

```diff
@@ -6,8 +6,9 @@
 
 
 @hookimpl
-def pyls_definitions(document, position):
-    definitions = document.jedi_script(position).goto_assignments()
+def pyls_definitions(config, document, position):
+    params = {k: v for k, v in config.plugin_settings('jedi_definition').items() if k}
```

Comment:
What does `if k` do here? Do you mean `if v` ? If you do, I think it should be `if v is not None`. Else if someone passes `False` for a property that jedi defaults to True, then the override won't be applied.

Reply:
@gatesn Fixed